app/controllers/cuisines_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CuisinesController < ApplicationController
+
+  # GET /cuisines.json
+  def index
+    render json: Cuisine.all
```

Comment:
Best to use a serializer e.g. `CuisineSerializer`. For example, If tomorrow someone added a sensitive field to `Cuisine` you wouldn't want it to leak to the client